libr/include/r_anal.h

```diff
@@ -1341,6 +1344,8 @@ R_API int r_anal_fcn_add_bb(RAnal *anal, RAnalFunction *fcn,
 		ut64 jump, ut64 fail, int type, RAnalDiff *diff);
 R_API bool r_anal_check_fcn(RAnal *anal, ut8 *buf, ut16 bufsz, ut64 addr, ut64 low, ut64 high);
 R_API void r_anal_fcn_update_tinyrange_bbs(RAnalFunction *fcn);
+R_API bool r_anal_fcn_tree_delete(RBNode **root, RAnalFunction *fcn);
```

Comment:
Imho datastruxture specific apis shouldnt be public. But we can refactor this out later

Reply:
Yes. And these `r_*.h` headers are too bloated. Many should not be exposed to end users. IMHO in many cases they are put there to be called by other modules. But because there is no boundary between (used by other modules) and (as APIs used by end users). This can cause much confuse and the unstable APIs are also very annoying.